.github/workflows/keep-alive.yml

```diff
@@ -0,0 +1,17 @@
+name: Keep
```

Comment:
@pavelzw What is this worklow for?

Reply:
GitHub Actions stops running cron workflows if there was no commit for 60 days. This will keep these actions alive.